docs/source/conf.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'TimeChart'
+copyright = '2019, hbui'
+author = 'Hai Bui'
+
+# The short X.Y version
+version = ''
+# The full version, including alpha/beta/rc tags
+release = '1.2.1'
```

Comment:
I would recommend to use the package version for `version` and `release` parameters. This way no one needs to remember to come in here and change the variables when a new version is released or when the code is updated.

Here is an example:
https://github.com/slaclab/pydm/blob/master/docs/source/conf.py#L69-L72